.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
add the .idea folder to your local gitignore so that it doesn't go up here and remove these files https://docs.github.com/en/github/getting-started-with-github/ignoring-files#excluding-local-files-without-creating-a-gitignore-file

Reply:
these .idea files are still here...